src/picongpu/include/plugins/PhaseSpace/PhaseSpaceFunctors.hpp

```diff
@@ -104,8 +104,8 @@ namespace picongpu
             int p_bin = int( rel_bin * float_X(num_pbins) );
 
             /* out-of-range bins back to min/max */
-            p_bin >= 0 ? /* do not change p_bin */ : p_bin=0;
-            p_bin < num_pbins ? /* do not change p_bin */ : p_bin=num_pbins-1;
+            p_bin >= 0 ? p_bin /* do not change p_bin */ : p_bin=0;
+            p_bin < num_pbins ? p_bin/* do not change p_bin */ : p_bin=num_pbins-1;
 
```

Comment:
Is there are good reason, why you use this method instead of an classicl "if"? Furthermore if clang doesn't like emptyness between `?` and `:`, `p_bin >= 0 ? true /* do not change p_bin */ : p_bin=0;` should also work? This way it looks like, something happends to p_bin, but that is not the case....